src/components/Icon/iconMapping.ts

```diff
@@ -5,6 +5,7 @@ import IconActionDelete from './components/IconActionDelete';
 import IconActionDownload from './components/IconActionDownload';
 import IconActionEdit from './components/IconActionEdit';
 import IconActionEmptyCircle from './components/IconActionEmptyCircle';
+import IconActionEmptyCircleSplit from './components/IconActionEmptyCircleSplit';
```

Comment:
Ignore [this comment](https://github.com/wellcometrust/design-system/pull/25/files#r789745436) I can see _EmptyCircle_ is the type and _Split_ is the variant